fendermint/vm/genesis/src/lib.rs

```diff
@@ -40,6 +40,9 @@ pub struct Genesis {
     /// highest possible fidelity when we are deriving a genesis file in IPC,
     /// where the parent subnet tracks collateral.
     pub validators: Vec<Validator<Collateral>>,
+    /// List of contract addresses that are allowed to deploy contracts in the subnet. If
+    /// empty, indicates there is no restriction on who can deploy contracts
+    pub contract_deployers: Vec<SignerAddr>,
```

Comment:
Let's roll back the genesis changes here, and instead have users who want to use this feature **fork Fendermint** and customize genesis within init. Rationale: there will be plenty of customizations possible down the line, and it won't scale to expose them all through genesis. In the future, these will be Wasm modules.

Reply:
But for now, how are they going to config this?

Reply:
that means we put the addresses in the source code?